website/routes.py

```diff
@@ -122,6 +123,7 @@ def get_globals():
         'recaptcha_site_key': settings.RECAPTCHA_SITE_KEY,
         'custom_citations': settings.CUSTOM_CITATIONS,
         'osf_support_email': settings.OSF_SUPPORT_EMAIL,
+        'wafflejs_url': '{api_domain}{waffle_url}'.format(api_domain=settings.API_DOMAIN[:-1] if settings.API_DOMAIN[-1] == '/' else settings.API_DOMAIN, waffle_url=reverse('wafflejs'))
```

Comment:
Drive-by: You can do `settings.API_DOMAIN.rstrip('/')` to strip the trailing `/` if it exists.